EuclideanGeometry/Foundation/Axiom/Linear/Ray_ex2.lean

```diff
@@ -15,30 +15,30 @@ theorem every_pt_lies_on_seg_of_source_and_target_lies_on_seg {seg₁ seg₂ : S
 
 /- Given two segments $seg_1$ and $seg_2$, if the source and the target of $seg_1$ both lie in the interior of $seg_2$, and if $A$ is a point on $seg_1$, then $A$ lies in the interior of $seg_2$. -/
 theorem
-every_pt_lies_int_seg_of_source_and_target_lies_int_seg : sorry := sorry
+every_pt_lies_int_seg_of_source_and_target_lies_int_seg {seg₁ seg₂ : Seg P}(h: seg₁.source LiesInt seg₂∧seg₁.target LiesInt seg₂){A : P}(ha: A LiesOn seg₁): A LiesInt seg₂:= sorry
```

Comment:
1. For beauty, please add space between each condition and before and after`:` or other infixes. e.g.
```
every_pt_lies_int_seg_of_source_and_target_lies_int_seg {seg₁ seg₂ : Seg P} (h : seg₁.source LiesInt seg₂ ∧ seg₁.target LiesInt seg₂) {A : P} (ha : A LiesOn seg₁) : A LiesInt seg₂ := sorry
```
2. We usually do not use `∧` in condition of a theorem, please split it into 2 conditions.